### DISCUSSION TITLE
Support for Contextual Routing in Remix

### DISCUSSION THREAD

**Discussion by prince272** [remix-run/remix#4005] [Proposals]
Does Remix support Contextual Routing

**Comment by sergiodxa** [comment]
~What do you mean by Contextual Routing?~

Just found what that is, not out of the box, but there's probably a way since it uses React Router and it's possible with it.

**Reply by prince272** [reply]
https://dev.to/toomuchdesign/contextual-routing-and-modal-routes-in-next-js-18fn

But how?
